### ISSUE TITLE
Fails to build with Sphinx 4: AttributeError: 'Sphinx' object has no attribute 'add_stylesheet'

### ISSUE THREAD

**Issue by audreytoskin** [jwilk/python-djvulibre#15]
[Sphinx 4.0](https://www.sphinx-doc.org/en/master/changes.html#release-4-0-0-released-may-09-2021) was released in May, and in the Fedora Project we've been testing and updating all dependent Python packages against the new major version of Sphinx. Including python-djvulibre :slightly_smiling_face:.

For your library, I believe the only backwards-incompatible change in Sphinx has to do with the `add_stylesheet` method, which is getting replaced by `add_css_file`. While building the current unpatched python-djvulibre 0.8.6 release as an RPM, I get this error:

```
Exception occurred:
  File "/builddir/build/BUILD/python-djvulibre-0.8.6/doc/api/conf.py", line 58, in setup
    app.add_stylesheet('docutils-math.css')
AttributeError: 'Sphinx' object has no attribute 'add_stylesheet'
```

Fortunately, seems like it's a straightforward drop-in replacement. I edited the files in the source tarball to replace all the old references to the old `add_stylesheet`, like this:

```
find ./ -type "f" -iname "*.py" -exec sed -E 's/add_stylesheet/add_css_file/g' --in-place '{}' ';'
```

And python-djvulibre seemed to build without issue after that.

**Comment by jwilk** [comment]
I believe I've fixed this in 07b63cd59c4f5bf647e2054b1c21d996a3ebd232.

I'll make a new release in a few days.

**Comment by jwilk** [comment]
Fixed in [0.8.7](https://github.com/jwilk/python-djvulibre/releases/tag/0.8.7).
